connectivity/FEATURE_BLE/libraries/TARGET_CORDIO_LL/stack/controller/sources/common/chci/chci_tr.c

```diff
@@ -174,9 +173,7 @@ static void chciTrHwError(uint8_t code)
 static void chciTrRead(uint16_t len, uint8_t *pData)
 {
   WSF_ASSERT(len <= LHCI_MAX_RD_BUF_LEN);
-#if (CHCI_TR_UART == 1)
   PalUartReadData(PAL_UART_ID_CHCI, pData, len);
```

Comment:
why are guards removed? is PalUartReadData a nop now?

Reply:
Hm, I still have the guard in my local copy.

Reply:
The whole function is guarded.

Reply:
The whole function is guarded now.